src/client/pages/OfferNew/Checkout/CheckoutContent.tsx

```diff
@@ -38,19 +39,25 @@ const StartDateWrapper = styled.div`
   margin-bottom: 2rem;
 `
 
+const StartDateLabel = styled.p`
```

Comment:
I also like this syntax better than the `styled('p')` (which I guess is older, or?) but maybe we should at least be consistent within files and use the same syntax everywhere, and in this file all other `styled` components are declared with the `('p')`.

Reply:
updated